src/angular/tiles/tile.component.html.ts

```diff
@@ -1,5 +1,7 @@
 export const template = `
-<ng-content select="sdc-tile-header"></ng-content>
-<ng-content select="sdc-tile-content"></ng-content>
-<ng-content select="sdc-tile-footer"></ng-content>
+<div [attr.data-tests-id]="testId">
```

Comment:
Instead of adding wrapper in HTML, you can use @HostBinding('attr.data-tests-id') in the component.